src/views/CommercialView.vue

```diff
@@ -102,11 +101,10 @@ onMounted(() => {
   if (props.unitId) {
     fetchCommercialDetails();
   } else {
-    console.warn('unitId fehlt – keine Daten können geladen werden.');
     toast.add({
       severity: 'warn',
-      summary: 'Ungültige ID',
-      detail: 'Gewerbe konnte nicht geladen werden, da keine ID übergeben wurde.',
+      summary: 'Invalid ID',
```

Comment:
Please use the i18n translation instead of changing the text directly in the code.
Here is an example: https://github.com/remsfal/remsfal-frontend/blob/bd1e09bad796e74ceb9e56c58deb6019ccaa4e19/src/views/AccountSettingsView.vue#L14

and here how to use: https://github.com/remsfal/remsfal-frontend/blob/bd1e09bad796e74ceb9e56c58deb6019ccaa4e19/src/views/AccountSettingsView.vue#L579